src/app/checkout/step-1/step-1.tpl.html

```diff
@@ -172,7 +172,7 @@ <h4 class="panel-title  border-bottom-small  visible" translate>Mailing Address<
                 <select class="form-control  form-control-subtle"
                         name="addressCountry" required
                         ng-model="$ctrl.donorDetails['mailing-address']['country-name']"
-                        ng-options="v['display-name'] | uppercase as v['display-name'] for v in $ctrl.countries"
+                        ng-options="v['display-name'] | uppercase as v['display-name'] for v in $ctrl.countries | orderBy:'\'display-name\''"
```

Comment:
Does this need to be double quoted? It just looks odd. I'm assuming you had an issue with passing the variable to orderBy since it has a dash.